polyfills/navigator/sendBeacon/polyfill.js

```diff
@@ -0,0 +1,33 @@
+(function(root) {
+
+	function sendBeacon(url, data) {
+		var xhr = ('XMLHttpRequest' in window) ? new XMLHttpRequest() : new ActiveXObject('Microsoft.XMLHTTP');
+		xhr.open('POST', url, false);
+		xhr.setRequestHeader('Accept', '*/*');
+		if (typeof data === 'string') {
+			xhr.setRequestHeader('Content-Type', 'text/plain;charset=UTF-8');
+			xhr.responseType = 'text/plain';
+		} else if (Object.prototype.toString.call(data) === '[object Blob]') {
+			if (data.type) {
+				xhr.setRequestHeader('Content-Type', data.type);
+			}
+		}
+		xhr.send(data);
+		return true;
+	}
+
+	if (!('sendBeacon' in navigator)) {
+		navigator.sendBeacon = sendBeacon;
```

Comment:
@JakeChampion what do you think about the global ref to `navigator` here?  I think it should be:

```
root.navigator.sendBeacon = sendBeacon;
```

And it should probably be in a final `else` on the UMD below, so it does not expose the global if in commonJS or AMD environments (though that's irrelevant to us)